tests/unit/plugins/test_maven_plugin.py

```diff
@@ -224,20 +226,22 @@ def test_settings_proxy(
           <proxies>
             <proxy>
               <id>{expected_protocol}_proxy</id>
-              <active>true</active>
               <protocol>{expected_protocol}</protocol>
               <host>my-proxy-host</host>
               <port>3128</port>
               <nonProxyHosts>{non_proxy_hosts}</nonProxyHosts>
+              <username>username</username>
+              <password>hunter7</password>
```

Comment:
We can also have a test with empty credentials to make sure it's generating the right output.